packages/audio-native/scripts/build.mjs

```
@@ -0,0 +1,38 @@
+    // (Stream Deck / dev plugin holding the DLL). Everything else (compiler
+    // errors, missing toolchain, bad gyp config, …) must surface so real
+    // build regressions aren't silently papered over.
+    const err = /** @type {NodeJS.ErrnoException} */ (error);
+    const message = String(err?.message ?? "");
+    const isLockOrPermissionError =
+      err?.code === "EBUSY" ||
+      err?.code === "EPERM" ||
+      /in use by another process|being used by another process|permission denied/i.test(message);
+
+    if (isLockOrPermissionError && existsSync(addonPath)) {
+      console.warn("node-gyp rebuild failed (file may be locked by a running process). Using existing native addon.");
```

ESLint

[error] 'console' is not defined.

(no-undef)